website/content/docs/nia/network-drivers/terraform-cloud.mdx

```diff
@@ -12,6 +12,8 @@ description: >-
   which is available with <strong>Consul Enterprise</strong>.
 </EnterpriseAlert>
 
+-> **Beta:** The integration with the HashiCorp managed service version of Terraform Cloud is currently only available in Consul-Terraform-Sync v0.4.0-beta.
```

Comment:
Suggestion only if you like - thinking about the perspective of someone who is newly interested in CTS (i.e. someone who wasn't aware of our prior decision to distinguish HC vs. self managed and then first prioritize self-managed), it could be helpful for these people to add something to clarify that the self-managed service version of Terraform Cloud was already available.